compute/cloud-client/src/main/java/compute/CreateInstanceFromTemplate.java

```diff
@@ -55,13 +62,46 @@ public static void createInstanceFromTemplate(String projectId, String zone, Str
       String instanceTemplateUrl)
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
 
-    try (InstancesClient instancesClient = InstancesClient.create()) {
+    try (InstancesClient instancesClient = InstancesClient.create();
+        InstanceTemplatesClient instanceTemplatesClient = InstanceTemplatesClient.create()) {
+
+      String instanceTemplateName = instanceTemplateUrl.substring(
```

Comment:
Is there not a helper method to get the name?

Reply:
I checked and looks like there's no need to pass the template URL here, hence refactored to just pass the template name. Thanks.